dlt/load/load.py

```diff
@@ -381,6 +436,18 @@ def complete_package(self, load_id: str, schema: Schema, aborted: bool = False)
             f"All jobs completed, archiving package {load_id} with aborted set to {aborted}"
         )
 
+    def update_loadpackage_info(self, load_id: str) -> None:
+        # update counter we only care about the jobs that are scheduled to be loaded
+        package_jobs = self.load_storage.normalized_packages.get_load_package_jobs(load_id)
```

Comment:
this is really costly operation... make sure we update counters only when job changed state

Reply:
this should be done correctly now, I am also updating it once after the loop exits, I hope this is good